src/hal/src/Kconfig

```diff
@@ -1,9 +1,25 @@
-menu "IMU sensor configuration"
```

Comment:
IMU sensor configuration was removed from here because a more complete version was used from the top-level `Kconfig` file and this file was unused until now.